.github/workflows/builds-and-tests.yml

```diff
@@ -163,7 +174,8 @@ jobs:
     # Code coverage (process and upload reports)
     - name: codecov
       if: matrix.codecov == 'YES'
-      uses: mfem/github-actions/upload-coverage@master
+      uses: mfem/github-actions/upload-coverage@testing/bernede1/codecov
```

Comment:
Change back?

Reply:
In a meeting, I’ll do the necessary. Good catch, I thought I had done that yesterday...

Reply:
It was a mistake to share the actions code AND to point to a branch instead of a tag / version (especially master).
The reason is now I have a breaking change (require lcov), and if I push it to master, it will break the previous workflow.
I have ideas about how to make that work: I will probably push a v.1.0 branch instead of master, and point to it.
I’ll do that next week along with the other changes needed.